app/lib/dqt/params/withdrawal.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Dqt
+  module Params
+    class Withdrawal
+      WITHDRAWN = "Withdrawn"
+
+      attr_reader :trainee, :params
+
+      def initialize(trainee:)
+        @trainee = trainee
+        @params = build_params
+      end
+
+      def to_json(*_args)
+        params.to_json
+      end
+
+    private
+
+      def build_params
+        {
+          "ittProviderUkprn" => trainee.hesa_student&.ukprn,
```

Comment:
This needs to come from `trainee.provider.ukprn`. The `hesa_students` table is just a temporary thing (hopefully). Not all trainees come through HESA either.

Reply:
oops, not sure where I got that idea from. Changed...